File: Data Structures & Algorithms/duplicate-integer/submission-0.cs
public class Solution {
    public bool hasDuplicate(int[] nums) {
        for (int i = 0; i < nums.Length; i++) {
            for (int b = i+1; b < nums.Length; b++) {
                if (nums[i] == nums[b]) {
                    return true;
                }
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/house-robber/submission-0.py
class Solution:
    
    
    def rob(self, nums: List[int]) -> int:
        while len(nums) > 3:
            first = nums[0] + nums[2]
            second = max(nums[0] + nums[3], nums[1] + nums[3])
            nums = [first, second] + nums[4:]
            print(nums)
        if (len(nums) == 1):
            return nums[0]
        elif (len(nums) == 2):
            return max(nums[0], nums[1])
        else:
            return max(nums[0] + nums[2], nums[1])

